### ISSUE TITLE
Windows: window accents on borders are not drawn

### ISSUE THREAD

**Issue by whaaaley** [microsoft/vscode#165399]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.73.0
- OS Version: Win11 22h2

Steps to Reproduce:

1. Open vscode
2. Set window in focus

---

In windows there's a setting to enable window accents on borders. I use this to keep track of what window is currently active. But in vscode there is no border. I use this for a11y reasons. It helps me track active windows visually.

Could active window border be supported? Thanks

![image](https://user-images.githubusercontent.com/7419947/199798617-2947fd0c-5248-48e7-9912-70f43571f22b.png)
![image](https://user-images.githubusercontent.com/7419947/199798982-9c297b54-1a52-42ac-9690-4bcfe57f260f.png)

**Comment by bpasero** [comment]
This is probably due to us disabling window native controls on Windows to draw a custom menu. As a workaround you can configure: `window.titleBarStyle: "native"` and it should be fine.

Marking as Electron issue given upstream issues such as https://github.com/electron/electron/issues/35289

I think it would be great if the border could be drawn natively, because we had to disable our support for drawing a custom window border color as it was not working reliably.

**Comment by rezelute** [comment]
Thanks for the comment, this still doesnt work for me, have i added it in the wrong place? (Using Win10)
```
"workbench.colorCustomizations": {
         "tab.activeBackground": "#ee3eb344",
         "window.titleBarStyle": "native",
         "window.activeBorder": "#b6b6b6",
         "window.inactiveBorder": "#797979"
     },
```
Just an aside, the reason why i use it is because its easy to distinguish between vscode windows when they are close to each other.
![image](https://user-images.githubusercontent.com/13378850/201894624-5c4f8f8b-6e5d-44d1-9c5f-f40f162bda4c.png)

**Comment by bpasero** [comment]
Support for window border was pulled out for Windows beginning with `1.71.x` version. We enabled window control overlays and window borders do not work well in combination. Besides, borders were never fully working depending on window zoom levels and screen DPI. As such, there is https://github.com/microsoft/vscode/issues/153222 for supporting a native window border from the OS that is not done by CSS from inside the window.

**Comment by deepak1556** [comment]
@rezelute `"window.titleBarStyle": "native"` is a top-level setting, you should move it outside `workbench.colorCustomizations`.

Merging to https://github.com/microsoft/vscode/issues/153222

**Comment by rezelute** [comment]
Ah great thanks for letting me know, thats working now and is doing the job thanks 🙏
